RELEASES.md

```diff
@@ -2,54 +2,128 @@
 - `Size<f32>.zero()` is now `Size::<f32>::ZERO`
 - `Point<f32>.zero()` is now  `Point::<f32>::ZERO`
-- renamed `taffy::node::Taffy.new_leaf()` -> `taffy::node::Taffy.new_leaf_with_measure()`
-- removed the public `Number` type; a more idiomatic `Option<f32>` is used instead
+- `Size::undefined()` is now `Size::NONE`
+
+#### Removals
+
+- Removed `taffy::forest::Forest`. `taffy::node::Taffy` now handles it's own storage using a slotmap (which comes with a performance boost up to 90%).
+- Removed `taffy::number::Number`. Use `Option<f32>` is used instead
   - the associated public `MinMax` and `OrElse` traits have also been removed; these should never have been public
-- `Sprawl::remove` now returns a `Result<usize, Error>`, to indicate if the operation was sucessful, and if it was, which ID was invalidated.
-- renamed `taffy::forest::Forest.new-node(..)` `taffy::forest::Forest.new_with_children(..)`
-- renamed `taffy::node::Taffy.new-node(..)` -> `taffy::node::Taffy.new_with_children(..)`
-- renamed `taffy::style::Style` -> `taffy::style::FlexboxLayout` to more precicely indicate its purpose
-- renamed `taffy::Error` -> `taffy::error::TaffyError`
-- `taffy::Taffy::remove_child_at_index`, `taffy::Taffy::replace_child_at_index`, and `taffy::Taffy::child_at_index` now return `taffy::InvalidChild::ChildIndexOutOfBounds` instead of panicing
-- `taffy::Node` is now unique only to the Taffy instance from which it was created.
-- `taffy::error::InvalidChild` is now `taffy::error::TaffyError`
-- `taffy::error::InvalidNode` has been removed and is now just a branch on the `TaffyError` enum
-- `taffy::forest::Forest` has been merged into `taffy::node::Taffy` for a performance boost up to 90%
-- `Size::undefined()` has been removed, use `Size::NONE` instead.
-- `Taffy.compute_layout()` now takes `Size<AvailableSpace>` instead of `Size<Option<f32>>`. If you were previously passing `Size::NONE` to this function, you will now need to pass `Size::MAX_CONTENT`
-- Measure functions now have an additional `available_space` parameter which indicates the size of the parent or a min/max-content sizing constraint.
-- Added `Size::MIN_CONTENT` and `Size::MAX_CONTENT` constants. In many cases, you will want to replace `Size::NONE` with `Size::MAX_CONTENT`.
 
-### 0.2.0 Fixed
+### Fixes
+
+Miscellaneous correctness fixes which align our implementation with Chrome:
 
-- Performance with deep hierarchies is greatly improved. It is now comparable to that of shallow hierarchies with the same number of nodes.
 - nodes can only ever have one parent
 - fixed rounding of fractional values to follow latest Chrome - values are now rounded the same regardless of their position
 - fixed computing free space when using both `flex-grow` and a minimum size
 - padding is now only subtracted when determining the available space if the node size is unspecified, following [section 9.2.2 of the flexbox spec](https://www.w3.org/TR/css-flexbox-1/#line-sizing)
```

Comment:
Nit: Can we capitalize those for consistency?

Reply:
Fixed